python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -33,20 +33,31 @@
 
 _stored_Interface = None
 
+_eval_cache = {}
+
 def setProxyInterface(my_interface):
+    ## Set the proxy interface, not strictly needed for Ganga.GPI but good practice as we move to 'ganga'
```

Comment:
If these are supposed to be docstrings then they should be """triple double quoted""".